### PR TITLE
Add field explanations for checkboxes

### PR DESCRIPTION
**Pull request by mattandrews**

### CODE REVIEW

File: views/pages/apply/reaching-communities-startpage.njk [hunk 1]
```diff
@@ -64,7 +64,7 @@
         <div class="u-constrained-content s-prose">
             <h2 class="t2 t--underline accent--pink">Other ways to apply</h2>
             <p>
-                We’re available every day from 5am to midnight to discuss
+                We’re available every day from 9am to 5pm to discuss
```
Comment (davidrapson) [inline views/pages/apply/reaching-communities-startpage.njk]: Good catch. Should this be something like "We’re available Monday–Friday from 9am to 5pm to discuss". Week days vs. everyday?

Comment (mattandrews) [reply views/pages/apply/reaching-communities-startpage.njk]: Good call, just changed.

**Comment by mattandrews** [comment]
Added some changes to highlight required/optional fields:

image

**Comment by davidrapson** [comment]
👍

**Pull request merged**